modules/services/xidlehook.nix

```
@@ -23,13 +23,14 @@ let
     ${concatStringsSep " " (notEmpty [
       "${cfg.package}/bin/xidlehook"
       (optionalString cfg.once "--once")
+      (optionalString cfg.detect-sleep "--detect-sleep")
       (optionalString cfg.not-when-fullscreen "--not-when-fullscreen")
       (optionalString cfg.not-when-audio "--not-when-audio")
       timers
     ])}
   '';
 in {
-  meta.maintainers = [ maintainers.dschrempf ];
+  meta.maintainers = with maintainers; [ dschrempf bertof ];
```

```suggestion
  meta.maintainers = [ maintainers.dschrempf hm.maintainers.bertof ];
```
HM's maintainer list is conserved in a different list.

---

Fixed.